src/test/java/com/apextrio/peak/PeakApplicationTests.java

```diff
@@ -1,17 +1,319 @@
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	public void testNewTeamNoAuth() {
+		try {
+			this.mvc.perform(get("/newteam?resortId=2")).andExpect(status().is3xxRedirection()).andExpect(redirectedUrl("http://localhost/login"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	@WithUserDetails("NJCrain")
+	public void testMyProfile() {
+		try {
+			this.mvc.perform(get("/myProfile")).andExpect(status().isOk()).andExpect(view().name("my_profile"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	public void testMyProfileNoAuth() {
+		try {
+			this.mvc.perform(get("/myProfile")).andExpect(status().is3xxRedirection()).andExpect(redirectedUrl("http://localhost/login"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	//---------------------------------
+	//TESTING FOR APPLICATION ENDPOINTS
+	//---------------------------------
+
+	@Test
+	@WithUserDetails("NJCrain")
+	public void testPostResortsId() {
+		try {
+			this.mvc.perform(post("/resorts/2?name=A Name&description=test&difficulty=3&capacity=5&dateGoing=2019-02-10T08:00")).andExpect(status().is3xxRedirection()).andExpect(redirectedUrlPattern("/teams/*"));
+		} catch (java.lang.Exception e) {
+			System.out.println(e);
+		}
+	}
+
+	@Test
+	public void testPostResortsIdNoAuth() {
+		try {
+			this.mvc.perform(post("/resorts/2?name=A Name&description=test&difficulty=3&capacity=5&dateGoing=2019-02-10T08:00")).andExpect(status().isForbidden());
+		} catch (java.lang.Exception e) {
+			System.out.println(e);
+		}
+	}
+
+	@Test
+	public void testGetResortId() {
+		try {
+			this.mvc.perform(get("/resorts/4")).andExpect(content().contentType(MediaType.APPLICATION_JSON_UTF8_VALUE)).andExpect(status().isOk()).andDo(print());
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	@WithUserDetails("NJCrain")
+	public void testPostTeamsId() {
+		try {
+			this.mvc.perform(post("/teams/3")).andExpect(status().is3xxRedirection()).andExpect(redirectedUrl("/teams/3"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	public void testPostTeamsIdNoAuth() {
+		try {
+			this.mvc.perform(post("/teams/3")).andExpect(status().isForbidden()).andExpect(redirectedUrl("/error"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	@WithUserDetails("NJCrain")
+	public void	testDeleteTeamId() {
+		try {
+			this.mvc.perform(delete("/teams/2")).andExpect(status().is3xxRedirection()).andExpect(redirectedUrl("/teams/2"));
+		} catch (java.lang.Exception e) {
+			System.err.println(e);
+		}
+	}
+
+	@Test
+	@WithUserDetails("jasonb315")
+	public void	testDeleteTeamIdLastUser() {
+		try {
+			this.mvc.perform(delete("/teams/2")).andExpect(status().is3xxRedirection()).andExpect(redirectedUrl("/"));
```

Comment:
I'd add one more perform here to get "/teams/2" after deleting it and confirm that that gives a 404.